client/src/assets/scss/_custom.scss

```diff
@@ -1273,6 +1273,7 @@ input::placeholder {
       .popup-heading-wrapper {
         height: 48px;
         text-align: center;
+        margin-right: 50px;
```

Comment:
Why do we need this?

Reply:
The issue was about the text that was not well placed and when I looked at the image I found that the text warning was not in the center because of that warning icon that comes first
![Screenshot from 2022-06-29 15-22-59](https://user-images.githubusercontent.com/32199791/176447286-5b9e0436-ff92-4f82-8ba2-31847d2bfeed.png)

Reply:
> @Diama1
> 
> The comment said `Popup text is different:`
> 
> ![Selection_879](https://user-images.githubusercontent.com/17882245/176448552-1b9c631a-cf58-44da-b58a-ca8bcc228777.png)
> 
> It's not about margin. Please revert this change and deal with the text itself.

Sure